ee/vellum_ee/workflows/display/nodes/base_node_display.py

```diff
@@ -272,14 +272,34 @@ def _serialize_attributes(self, display_context: "WorkflowDisplayContext") -> Js
 
     def serialize_generic_fields(self, display_context: "WorkflowDisplayContext") -> JsonObject:
         """Serialize generic fields that are common to all nodes."""
-        return {
+        result: JsonObject = {
             "display_data": self.get_display_data().dict(),
             "base": self.get_base().dict(),
             "definition": self.get_definition().dict(),
             "trigger": self.serialize_trigger(),
             "ports": self.serialize_ports(display_context),
         }
 
+        # Only include should_file_merge if there are custom methods defined
+        try:
+            node_class = self.__class__.infer_node_class()
+
+            # Exclude BaseNode methods such as Ports, Trigger
+            base_node_methods = set(BaseNode.__dict__.keys())
```

Comment:
[blocking] I don't think we want this piece as it means if our custom node overrides the `run` method, it gets skipped